### PR TITLE
Feature/buckets resource

### LINKED ISSUE #6: Add bucket resource

### PR DESCRIPTION
**Pull request by FredPi17**
Creation, updating and deletion of a bucket is running. 
Closes #6

### CODE REVIEW

File: website/influxdbv2.erb [hunk 1]
```diff
@@ -9,6 +9,15 @@
         <li<%= sidebar_current("docs-influxdbv2-index") %>>
           <a href="/docs/providers/influxdbv2/index.html">InfluxDB V2 Provider</a>
         </li>
+
+        <li<%= sidebar_current("docs-do-datasource") %>>
```
Comment (ppff) [inline website/influxdbv2.erb:13]: Should be `docs-influxdbv2-resource`

File: website/influxdbv2.erb [hunk 2]
```diff
@@ -9,6 +9,15 @@
...
+          <a href="#">Resources</a>
+          <ul class="nav nav-visible">
+            <li<%= sidebar_current("docs-influxdbv2-resource-bucket") %>>
+              <a href="/docs/r/bucket.html.md">bucket</a>
```
Comment (ppff) [inline website/influxdbv2.erb:17]: Should be `/docs/providers/influxdbv2/r/bucket.html`

File: website/docs/r/bucket.html.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+##Argument Reference
+
+The following arguments are supported: 
+
+* ``name`` (Required) The name of the bucket.
+
+* ``description`` (Optional) The description of the bucket.
+
+* ``org_id`` (Optional) The organization id the bucket is linked.
```
Comment (ppff) [inline website/docs/r/bucket.html.md:46]: Are you sure it's not required? What happens if no org_id is set?

File: website/docs/r/bucket.html.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+* ``retention_rules`` (Optional) Retention rules that affects the bucket.
```
Comment (ppff) [inline website/docs/r/bucket.html.md:48]: Retention rules are required in the API doc. What happens if we don't set them in terraform?

File: website/docs/r/bucket.html.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+description: |-
+  The influxdbv2_bucket resource allows an InfluxDB to create, update and delete buckets.
+---
+
+## Example Usage
+
+###Create a new bucket
+```hcl
+resource "influxdbv2_bucket" "initial" {
+    name = "Le nouveau bucket"
```
Comment (ppff) [inline website/docs/r/bucket.html.md:14]: No french in here!

File: website/docs/r/bucket.html.md (hunk 3 above)
Comment (FredPi17) [reply website/docs/r/bucket.html.md:46]: Org_id is not required in the API documentation. I thing if org_id is not set, the bucket is accessible for all orgs.

File: website/docs/r/bucket.html.md (hunk 4 above)
Comment (FredPi17) [reply website/docs/r/bucket.html.md:48]: Oh it's an error for me, i set it to required now.

**Comment by FredPi17** [comment]
[ ] Add debug logging and contextualize errors (example: https://github.com/terraform-providers/terraform-provider-digitalocean/blob/master/digitalocean/resource_digitalocean_database_cluster.go#L196) 

-> I added contextualized errors, but debug logging is already present in the client for each route called.

**Pull request merged**